snapshots/devmapper/config.go

```diff
@@ -82,6 +85,10 @@ func (c *Config) parse() error {
 		return errors.Wrapf(err, "failed to parse base image size: '%s'", c.BaseImageSize)
 	}
 
+	if c.FileSystemType == "" {
```

Comment:
Do we need to validate the fstype when init?

Reply:
Added a validation in validate() function. Inside parse() we set default fs type to ext4 if config for fs type is missing.